pages/affinity-diagramming.md

```diff
@@ -11,7 +11,7 @@ A way of finding themes in collections of ideas, quotes, or observations.
 
 ## Reasons to use it
 
-Quick and collaborative approach for drawing insights from qualitative data.
+To draw out insights from qualitative data quickly and collaboratively.
```

Comment:
@jenniferthibault Yeah, sorry, just forgot that this time.

Who to flag is probably an internal-to-18F thing to figure out, right? Or do you think it's worth adding a handle for print card things in the contrib file? It seems a bit incongruous to me since we don't list people for everything else (and it would be unwieldy to keep all those up to date), but we can if you'd rather.